sql-select/two-kilmers.sql

```diff
@@ -0,0 +1,5 @@
+select "actorId",
+      "firstName"
+      from "actors"
+      where 'Kilmer' = "lastName"
```

Comment:
Usually you would swap these and put the column name first. But that's just convention. It works either way.